sylt-parser/src/statement.rs

```diff
@@ -199,25 +203,56 @@ pub fn statement<'t>(ctx: Context<'t>) -> ParseResult<'t, Statement> {
         // `use path/to/file as alias`
         [T::Use, ..] => {
             let (ctx, path) = path(ctx.skip(1))?;
-            let file = Identifier {
-                span: ctx.span(),
-                name: path,
+            let bare_name = path
+                .trim_start_matches("/")
+                .trim_end_matches("/");
+            let file = {
+                let parent = if path.starts_with("/") {
+                    ctx.root
+                } else {
+                    ctx.file.parent().unwrap()
+                };
+                // Importing a folder is the same as importing exports.sy
+                // in the folder.
+                parent.join(if path == "/" {
+                    format!("exports.sy")
+                } else if path.ends_with("/") {
+                    format!("{}/exports.sy", bare_name)
+                } else {
+                    format!("{}.sy", bare_name)
+                })
             };
-            let (ctx, file_alias) = match &ctx.tokens[ctx.curr..] {
+            let (ctx, ident) = match &ctx.tokens[ctx.curr..] {
                 [T::As, T::Identifier(alias), ..] => (
                     ctx.skip(2),
-                    Some(Identifier {
+                    Identifier {
                         span: ctx.skip(1).span(),
                         name: alias.clone(),
-                    })
+                    }
                 ),
                 [T::As, ..] => raise_syntax_error!(
-                    skip_until!(ctx, T::Newline),
+                    ctx.skip(1),
                     "Expected alias"
```

Comment:
Maybe this can be a bit clearer?
`Expected 'as' followed by an alias identifier`?

Reply:
Well, I think it already is a very clear error when seen in the wild.
But maybe `Expected alias identifier`?

Reply:
@sornas